build/build.js

```diff
@@ -83,15 +91,28 @@ function displayStats() {
 }
 
 
-
-async function fetchAndDisplayCharacter() {
+async function fetchAndDisplayCharacter(character) {
```

Comment:
do you want to pass character in as an argument here?  If do you want to fetch the character with `getCharacter`?  and assign that to a character variable?